Tribler/Core/Modules/versioncheck_manager.py

```diff
@@ -0,0 +1,48 @@
+from distutils.version import LooseVersion
+import json
+import logging
+from twisted.internet.task import LoopingCall
+from twisted.web.client import Agent, readBody
+from twisted.web.http_headers import Headers
+from Tribler.Core.Utilities.twisted_thread import reactor
+from Tribler.Core.simpledefs import NTFY_INSERT, NTFY_NEW_VERSION
+from Tribler.Core.version import version_id
+from Tribler.dispersy.taskmanager import TaskManager
+
+
+VERSION_CHECK_URL = 'https://api.github.com/repos/tribler/tribler/releases/latest'
+
+
+class VersionCheckManager(TaskManager):
+
+    def __init__(self, session):
+        super(VersionCheckManager, self).__init__()
+
+        self._logger = logging.getLogger(self.__class__.__name__)
+        self.session = session
+        self.version_check_task = None
+
+    def start(self, interval):
+        self.version_check_task = self.register_task("tribler version check", LoopingCall(self.check_new_version))
```

Comment:
minor thing: You don't really need to keep a persistent reference to the version check task as it's registered on the TaskManager